packages/server/src/db/user.ts

```diff
@@ -9,7 +9,7 @@ export class User {
   @Index()
   username: string
 
-  @Column('varchar')
+  @Column({ type: 'varchar', collation: 'nocase' })
```

Comment:
Made the password field case-insensitive.  This will help with uniqueness if we ever enforce that.  But more importantly here, it helps with user lookups by email, acknowledging that the user may not type in their password with the same casing every time.

Reply:
I'm guessing you meant "email" here not "password". But nice I like it 👌

Reply:
Yep!  I've edited the comment to avoid further confusion 🙃